src/lib/style/components/_access-requirement-list.scss

```diff
@@ -1,93 +1,123 @@
-.data-access-requirement-title {
-  margin-top: 30px;
-}
+.AccessRequirementList {
+  color: #515359;
 
-.requirement-container {
-  display: flex;
-}
+  &__modal {
+    padding: 15px;
+  }
 
-.register-text-link {
-  color: rgb(77, 85, 144);
-}
+  p {
+    margin-bottom: 0px;
+  }
 
-.terms-of-use-content {
-  margin-top: 20px;
-}
+  &__instruction {
+    font-size: 18px;
+    font-weight: bold;
+    text-transform: uppercase;
+  }
+  &__access {
+    margin-top: -10px;
+  }
 
-.button-container {
-  &.hide {
-    visibility: hidden;
+  &__title {
+    margin-left: -15px;
+    font-size: 23px;
+    font-weight: bold;
+  }
+  &__file {
+    margin-right: 10px;
+    color: #bbbbbc;
   }
-}
 
-.view-terms-button {
-  color: rgb(77, 85, 144);
-}
-.view-terms {
-  &.hide {
-    visibility: hidden;
+  .requirement-container {
+    display: flex;
+    align-items: center;
+    margin-top: 15px;
   }
-}
 
-.accept-button-container {
-  margin-top: 30px;
-  align-content: center;
-  justify-content: center;
-  text-align: center;
-}
+  &__register-text-link {
+    display: block;
+    color: $primary-action-color;
+    margin-top: 20px;
+    margin-bottom: 20px;
+  }
 
-.accept-button {
-  background-color: rgb(77, 85, 144);
-  padding: 10px 70px 10px 70px;
-  color: white;
-}
+  &__AccessRequirementList__signin .terms-of-use-content {
+    margin-top: 20px;
+  }
 
-.not-accept-button-container {
-  margin-top: 15px;
-  align-content: center;
-  justify-content: center;
-  text-align: center;
-  color: rgb(77, 85, 144);
-}
+  .button-container {
+    &.hide {
+      visibility: hidden;
+    }
+  }
 
-.self-sign-access-title {
-  margin-top: 20px;
-}
+  .view-terms-button {
+    color: rgb(77, 85, 144);
+  }
+  .view-terms {
+    &.hide {
+      visibility: hidden;
+    }
+  }
 
-.self-sign-access-certified-success-text {
-  &.hide {
-    visibility: hidden;
+  .accept-button-container {
+    margin-top: 30px;
+    align-content: center;
+    justify-content: center;
+    text-align: center;
   }
-}
 
-.self-sign-access-verified-success-text {
-  &.hide {
-    visibility: hidden;
+  .accept-button {
+    background-color: rgb(77, 85, 144);
+    padding: 10px 70px 10px 70px;
+    color: white;
+  }
+
+  .not-accept-button-container {
+    margin-top: 15px;
+    align-content: center;
+    justify-content: center;
+    text-align: center;
+    color: rgb(77, 85, 144);
+  }
+
+  .self-sign-access-title {
+    margin-top: 20px;
+  }
+
+  .self-sign-access-certified-success-text {
+    &.hide {
```

Comment:
per comment above I think if you'll add .hidden rather than .hide you will not need extra css definition